src/primaite/common/enums.py

```diff
@@ -168,20 +168,39 @@ class FileSystemState(Enum):
     RESTORING = 5
 
 
+class NodePropertyAction(Enum):
+    """Node property action types."""
+
+    NONE = 0
+    HARDWARE = 1
+    SOFTWARE = 2
+    SERVICE = 3
+    FILE_SYSTEM = 4
+
+
 class NodeHardwareAction(Enum):
     """Node hardware action."""
 
     NONE = 0
-    ON = 1
-    OFF = 2
+    TURN_ON = 1
```

Comment:
for my understanding - were all these just incorrect in the primaite codebase

Reply:
these were fine but i wanted to make it more clear what they represented

Reply:
Got it :D